lib/python/pyflyby/_util.py

```diff
@@ -11,6 +11,8 @@
 from   six                      import reraise
 import sys
 import types
+import ast
+import astunparse
```

Comment:
I don't think we should introduce a new dependency if possible. Can we use tidy-imports itself?

Reply:
Removed dependency on ast and astunparse